dev/automated_tests/flutter_test/print_user_created_ancestor_test.dart

```diff
@@ -12,7 +12,7 @@ void main() {
       CustomScrollView(
         slivers: <Widget>[
           SliverToBoxAdapter(child: Container()),
-        ]
+        ],
       )
```

Comment:
Is this also missing another trailing comma?

Reply:
Good catch! Actually my current lint rule is not enough aggressive to catch it. I'll try to catch more cases in another PR.